### ISSUE TITLE
Update bazelrc preset `--nolegacy_external_runfiles` for Bazel 8

### ISSUE THREAD

**Issue by hofbi** [bazel-contrib/bazel-lib#1016]
The `performance.bazelrc` preset defines https://github.com/bazel-contrib/bazel-lib/blob/d042d563c6a91f7e11f66c42c83429199bd3d5d9/.aspect/bazelrc/performance.bazelrc#L14. In [Bazel 8](https://github.com/bazelbuild/bazel/releases/tag/8.0.0) `--legacy_external_runfiles` (https://github.com/bazelbuild/bazel/issues/23574; now defaults to false).

Should we move the definition from `performance.bazelrc` into the Bazel 6 and Bazel 7 rc files? This would result in a small duplication.

**Comment by alexeagle** [comment]
Yes please, moving to bazel6 and bazel7 is the right solution. I don't think we want to create a `bazel_6_or_7` file, the duplication is not worth the complexity.
